eventbrite/requirements.txt

```diff
@@ -0,0 +1 @@
+autohive-integrations-sdk
```

Comment:
Is this SDK changed now so that we don't need to apply the version number or does it just work off the bat